src/views/Items/Item.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component, PropTypes} from 'react';
+import classNames from 'classnames';
+
+export default class Item extends Component {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    description: (props, propName, componentName) => {
+      if (props.children && props.description) {
```

Comment:
Nice.  That was easy :)  Let's pull this function out into a method on a props util so we can use it on all the components.  I'll also be adding a util soon to get spreadable props, I'll add it to your util after merge.